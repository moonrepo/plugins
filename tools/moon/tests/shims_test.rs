use proto_pdk_test_utils::*;

mod moon_tool {
    use super::*;

    #[cfg(not(windows))]
    generate_shims_test!("moon-test", ["moon"]);
}
