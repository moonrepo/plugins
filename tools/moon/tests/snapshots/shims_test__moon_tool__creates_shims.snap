---
source: tools/moon/tests/shims_test.rs
expression: "std :: fs ::\nread_to_string(sandbox.path().join(\".proto/shims/registry.json\")).unwrap()"
---
{
  "moon": {}
}
